assets/src/design-system/components/dropDown/menu/defaultListItem.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Checkmark } from '../../../icons';
+import { DROP_DOWN_ITEM } from '../types';
+import { ListItem } from './components';
+
+export const DefaultListItem = forwardRef(function DefaultListItem(
```

Comment:
I could envision a `const` and `function` with the name name causing an issue. You could just `export default forwardRef(DefaultListItem)` then define `function DefaultListItem({ ... })` above that way we can assign `propTypes` to it.

Reply:
Sounds good!